flow/connectors/core.go

```diff
@@ -70,6 +70,7 @@ type GetSchemaConnector interface {
 	GetColumns(ctx context.Context, schema string, table string) (*protos.TableColumnsResponse, error)
 	GetSchemas(ctx context.Context) (*protos.PeerSchemasResponse, error)
 	GetTablesInSchema(ctx context.Context, schema string, cdcEnabled bool) (*protos.SchemaTablesResponse, error)
+	GetColumnsTypeConversion() (*protos.ColumnsTypeConversionResponse, error)
```

Comment:
If we don't want to introduce this interface, we can just directly call a static function:

```
clickhouseconn.GetColumnsTypeConversion()
```